  <launch>
    <arg name="convo" />
    <arg name="exp" />
    <arg name="user" />

    <!-- Turtlesim Node-->
    <node pkg="turtlesim" type="turtlesim_node" name="sim"/>
    <!-- <node pkg="turtlesim" type="turtle_teleop_key" name="teleop" output="screen"/> -->
    <node pkg="gaze_turtle" type="turtlebot_teleop_key.py" 
          name="teleop"/>    

    <node name="turtle1_tf_broadcaster" pkg="gaze_turtle" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle1" />
    </node>
    <node name="turtle2_tf_broadcaster" pkg="gaze_turtle" type="turtle_tf_broadcaster.py" respawn="false" output="screen" >
      <param name="turtle" type="string" value="turtle2" /> 
    </node>
    <node pkg="gaze_turtle" type="turtle_tf_listener.py" 
          name="listener" output="screen"/>
    <node pkg="gaze_turtle" type="hmm.py" 
          name="hmm" args="$(arg convo) $(arg exp)" output="screen"/>
    <node pkg="gaze_turtle" type="robot_speech.py" 
          name="robot_speech" output="screen"/>
    <node pkg="gaze_turtle" type="logger.py" 
          name="poli_logger" args="$(arg user) $(arg exp)" output="screen"/>
   
 </launch>

